weave/scorers/base_scorer.py

```diff
@@ -16,7 +16,7 @@ class Scorer(Object):
         description="A mapping from column names in the dataset to the names expected by the scorer",
     )
 
-    def score(self, input: Any, target: Any, output: Any) -> Any:
+    def score(self, *, output: Any, **kwargs: Any) -> Any:
```

Comment:
This is more representative of the actual score func signature.  Some scorers don't take input, or may take other args

Reply:
agree, and `input` is not necessary an available col.